screens/Contact/ContactDetailScreen.js

```diff
@@ -64,19 +69,19 @@ const milestonesGridSize = windowWidth + 5;
 /* eslint-disable */
 let commentsFlatList,
   subAssiSelectizeRef,
-  geonamesSelectizeRef,
-  pplGroupsSelectizeRef,
+  geonamSelectizeRef,
+  pplGroupsSelectRef,
   sourcesSelectizeRef,
   groupsSelectizeRef,
   connectSelectizeRef,
   baptBySelectizeRef,
   coachedSelectizeRef,
-  baptizedSelectizeRef,
-  coachingSelectizeRef;
+  baptSeleRef,
+  coachiSelectizeRef;
```

Comment:
Why are all of these variables misspelled? If it was to shorten them, we should not do that. It is better for maintainability to keep variable names longer and more descriptive.

Reply:
Hi @cairocoder01, i have to cut the variables because of this eslint rule (.eslintrc.json): 

`"max-len": ["error", 100, 2, {
      "ignoreUrls": true,
      "ignoreComments": true,
      "ignoreRegExpLiterals": true,
      "ignoreStrings": true,
      "ignoreTemplateLiterals": true
    }],`

Some lines in the template are more than 100 length size. when i try to do the commit, the pre-commit eslint validation throw this error and i can't end the commit until y fix this "errors".

Reply:
I can use the /* eslint-disable */ and /* eslint-enable */ comments to ignore these situations. ¿What's your opinion about it?.

Reply:
Is it a matter of "upping" the length size to 120 or 160 or even 200 and then having it work?  I'd rather have readable variable names than enforce a strict line length.

Reply:
Yup, so i will go to increase the length of "max-len" rule to 200 and modify the variables names to be more readables.